### DISCUSSION TITLE
Why is Multiview "Native Only"?

### DISCUSSION THREAD

**Discussion by jpryne** [gfx-rs/wgpu#2390] [Q&A] [answered]
Is this feature intended for (native) VR/XR? Are there plans to extend this render feature to WebXR? Please correct me if I'm wrong in thinking this is an important feature for an optimal WebXR experience.

**Comment by kvark** [comment, marked as answer]
In order to make it available on the web, it has to be standardized first in https://github.com/gpuweb/gpuweb
